### ISSUE TITLE
ver 4.93 not working with Krypton on Win 7

### ISSUE THREAD

**Issue by SquigglyDog** [cubicle-vdo/xbmc-israel#401]
I have a beta 2 version of Krypton on a Win 7 machine. Israelive installs but will not function. Will paste a Kodi log tomorrow

**Comment by avigdork** [comment]
I tried it now on Win 10 and it run without a problem. 
It should run on Win 7 as well.

**Comment by SquigglyDog** [comment]
I get the error message "could not Load Settings" when I try to open Israelive. My Kodi.log is at http://pastebin.com/Nc1Lq7H1
Thanks for your patience

**Comment by SquigglyDog** [comment]
By the way it works fine for me on Linux Mint 18

**Comment by SquigglyDog** [comment]
I think I solved it. I notice in my log the entry "08:58:39 T:3964 WARNING: [Error 2] The system cannot find the file specified: u'C:\\Users\\HP PC\\AppData\\Roaming\\Kodi\\userdata\\addon_data\\plugin.video.israelive\\remoteSettings.txt'". This file remoteSettings.txt was in fact absent from the addon_data folder . I looked in my Linux PC and found that file along with about 10 other files missing from my Win 7 PC. So I copied the files from the Linux PC to  the Win 7 PC and Israelive seems to work fine now. Not sure why those files were not installed when I installed Israelive
